doc/api/inspector.md

```diff
@@ -0,0 +1,102 @@
+# Inspector
+
+> Stability: 1 - Experimental
+
+The `inspector` module provides an API for interacting with the V8 inspector.
+
+It can be accessed using:
+
+```js
+const inspector = require('inspector');
+```
+
+## Class: inspector.Session
+
+The `inspector.Session` class is a primary means of dispatching messages to
+the V8 inspector back-end and receiving message responses and notifications.
+
+### Constructor: new inspector.Session()
+<!-- YAML
+added: REPLACEME
+-->
+
+Create a new instance of the `inspector.Session` class. The resulted inspector
+session needs to be connected through [`session.connect()`][] before
+messages can be sent to the inspector backend.
+
+`inspector.Session` is is an [`EventEmitter`][] with the following events:
+
+### Event: 'inspectorNotification'
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Object} The notification message object
+
+Emitted when a notification from the V8 Inspector is received.
+
+### Event: INSPECTOR_PROTOCOL_METHOD
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Object} The notification message object
+
+Emitted when an inspector notification is recieved that has method field set
+to INSPECTOR_PROTOCOL_METHOD value.
```

Comment:
I think there’s a missing `the` before `INSPECTOR_PROTOCOL_METHOD`, and I’d put the latter in backticks